compiler/rustc_feature/src/unstable.rs

```diff
@@ -371,6 +371,8 @@ declare_features! (
     (unstable, async_for_loop, "1.77.0", Some(118898)),
     /// Allows using C-variadics.
     (unstable, c_variadic, "1.34.0", Some(44930)),
+    /// Allows the use of `#[cfg(<true/false>)]`.
+    (unstable, cfg_boolean_literal, "CURRENT_RUSTC_VERSION", Some(999999)),
```

Comment:
note to self: remember to update the tracking issue number here (and the the unstable-book) once the RFC is merged and the tracking issue is created.

Reply:
I created a tracking issue (#131204) since it seems like this will be merged soon